Revert "W-16431086 W-17577350: Document changes around compilation of connectors with Java 17" (#276)
mulesoft/docs-mule-sdk: modules/ROOT/pages/xml-sdk.adoc

Install and verify that the following tools are working in your local environment:

* Java Development Kit 8 (JDK 8) to compile and build your Java code.
+

> NOTE: The maximum supported version for XML SDK is JDK 17, however, you can use JDK 17 only for running your application. Compilation with XML SDK must be done with JDK 8.

Reviewer: ```suggestion
NOTE: The maximum supported version for XML SDK is JDK 17; however, you can use JDK 17 only for running your application. To compile with XML SDK, you must use JDK 8.
```
or preferably, if multiple sentences work with a note: " ... JDK 17. However, ... "
